src/logging.js

```diff
@@ -1,27 +1,42 @@
-function logInfo(message) {
-  if (typeof newrelic !== 'undefined') {
-    newrelic.addPageAction('INFO', { message }); // eslint-disable-line no-undef
+let loggingService = null;
```

Comment:
This file is interesting, in that it seems like these checks would be useful to any library configuring a logging service... does this belong in frontend-auth?

Reply:
Moved to https://github.com/edx/frontend-logging/pull/6